lib/src/Base/Common/TBB.cxx

```diff
@@ -85,6 +88,12 @@ TBB_init::TBB_init()
 #else
     pthread_mutex_init( &TBB_InstanceMutex_, NULL );
 #endif
+
+#if defined(OPENTURNS_HAVE_TBB) && defined(_OPENMP)
+    // we dont want threading at multiple levels (eg openblas)
+    omp_set_num_threads(1);
```

Comment:
how to reactivate `openmp` ?

Reply:
I did not add a method to do that

Reply:
So the number of openmp threads is fixed at compile time, no way for the user to benefit from blas parallelism even if he does not use the TBB/OpenMP combo. Maybe an entry in ResourceMap could be beneficial here...

Reply:
I am currently compiling your branch to perform some tests...

Reply:
A resourcemap entry would be unusable here as if we want it to be the default, it has to happen at the import.
But, this was just of proof of concept: maybe having this as the default behavior is not acceptable.
I propose to add a method, TBB::DisableOpenMPThreads() to to that,
and eventually TBB::EnableOpenMPThreads()

Reply:
@jschueller yes and apply these methods inside `computeParallelPDF(sample)` for example

Reply:
In my opinion OpenMP is used only in low-level classes, even in process-related algorithms (including kriging) the blocking factor is the discretization of the covariance, where OpenMP is of no help. For large problems, there is much more to gain using hmat than lapack, so let's keep the SSE2 part of OpenBLAS and forget about its OpenMP prallelism!